src/execution_plan/optimizations/convert_all_node_scans.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2018-2020 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Redis Labs Source Available License Agreement
+ */
+
+#pragma once
+
+#include "../ops/ops.h"
+#include "../execution_plan.h"
+
+void convertAllNodeScans(ExecutionPlan *plan);
```

Comment:
Add comments about this function
If this function limited only to all_node_scan->label_scan transformation I think it should be renamed, unless there is planning for the future

Reply:
Yeah, this is poorly named. What do you think of `promoteLabelFilters`?